src/engagement_db_to_analysis/generate_production_file.py

```diff
@@ -0,0 +1,7 @@
+from core_data_modules.traced_data.io import TracedDataCSVIO
+
+
+def generate_production_file(messages_td, production_csv_output_path):
+    production_keys = ["participant_uuid", "dataset", "text"]
+    with open(production_csv_output_path, "w") as f:
+        TracedDataCSVIO.export_traced_data_iterable_to_csv(messages_td, f, production_keys)
```

Comment:
I don't think this is quite right as it just outputs a list of messages. The production file is formatted to give people easy access to quotes they can use on the radio, e.g. "a woman from nairobi said...", which means it needs to list demographics next to each message

Reply:
Thanks, I'm working on it.